### ISSUE TITLE
Python could not be properly inititalized.

### ISSUE THREAD

**Issue by GoogleCodeExporter** [pyscripter/pyscripter#715]
```
An error message pops up every time I try to open PyScripter. It says "Python 
could not be properly initialized. We must quit."
I'm using Windows 8 (desktop). I downloaded Python 2.7.3 Windows Installer from 
python.org. Whould I download another versio of Python. Please reply soon!
```

Original issue reported on code.google.com by `poop52...@gmail.com` on 5 Jan 2013 at 2:52

**Comment by GringoA1** [comment]
I tried PyScripter-v2.6.0-Setup - it was not working. But link below is working:

http://sourceforge.net/projects/pyscripter/files/PyScripter-v2.6.0-x64-Setup.exe/download

**Comment by svremo** [comment]
when i saw this problem on my setup, it was due to the fact that I installed 64 bit python and using the pyscripter at http://sourceforge.net/projects/pyscripter/ that (i assume) is a 32 bit version.  With this i cannot make the pyscripter work.

From the pyscripter page at sourcefource I clicked "Files" on the tab and see a x64 version. Link is below (same as what GringoA1 did).
http://sourceforge.net/projects/pyscripter/files/PyScripter-v2.6.0-x64-Setup.exe/download

Thank you!

**Comment by iFowlian** [comment]
Installing pyscripter in my python 2.7 folder worked for me.

**Comment by abbyrose1986** [comment]
Yea, except most people don't want to always run it from command line. So, first time run it with sudo. Then go to Tools -> Create desktop entry. After that you have shortcut in launcher. Same thing when you update pycharm so the launcher points to the new version.﻿<a href="http://serialkeygeneratorfree.com/serial-windows-8-1-product-key-crack-free-download/">windows 8.1 key </a>

**Comment by pauljames** [comment]
Following worked for me:

python-3.4.4.amd64
PyScripter-v2.6.0-x64-Setup

on 64 bit windows 10.

NOTE: had to use earlier version of python.

**Comment by jhblan13** [comment]
Thank you Fahad Rahmani. I spent days on trying to get this to work. Your suggestion saved me! It works!

**Comment by Gettyk** [comment]
python-3.6.0b3-amd64 doesnt work with PyScripter

**Comment by BNNorman** [comment]
If you use sysinternals you can see PyScripter scanning for python 3.4 downwards. Shame it isn't updated to use the latest version. I've switched to PyCharm community edition because of this - I loved PyScripter - it fired up quicker than PyCharm.

**Comment by snord** [comment]
Again, comment  #8 is the answer! 
Thank your so much Fahad Rahmani, your knowledge and generosity is so very much appreciated!

**Comment by IainHaggis** [comment]
Pyscripter 3.2.1 does not work with Python 3.7.0a both are 32 bit. I get the error ""Python 
could not be properly initialized." Was running slightly older version successfully but for what ever reason the newer versions above won't run. I've tried uninstalling and reinstalling both Python and then Pyscripter 3 or 4 times to no avail. I'm running and old sloooow 64 bit machine (HP G62). As I said it did run with slightly earlier versions of Python and Pyscripter but newest version don't work.
Any suggestions?

**Comment by pyscripter** [comment]
It works with Python3.7.0a1 but not Python 3.7.0a4.  The C interface changed in between.  The issue is resolved in version control and will be released in PyScripter version 3.2.2.

**Comment by IainHaggis** [comment]
Many thanks, it was in fact 3.7.0a4 I was trying.
